DuckDuckGo/HomePageConfiguration.swift

```diff
@@ -32,39 +32,17 @@ class HomePageConfiguration {
     
     enum Component: Equatable {
         case navigationBarSearch(fixed: Bool)
-        case centeredSearch(fixed: Bool)
-        case extraContent
         case favorites
         case padding
```

Comment:
Can we delete the unused `padding` component and the associated `PaddingHomeViewSectionRenderer`?